object_hashes.d.ts

```diff
@@ -0,0 +1,9990 @@
+declare namespace RageEnums.Hashes {
+    const enum Object {
+        gate3_l = 3027077612,
```

Comment:
Again I would say that we should stick to uppercase constant names - even though there are A LOT of names to changes this on.